client/src/mocks/handlers.ts

```diff
@@ -15,13 +15,6 @@ export const handlers = [
   rest.delete(`${API_PATHS.bff}/product/:id`, (req, res, ctx) => {
```

Comment:
Hello Eduard. Looks like you did the task in the wrong repositories. You had to create the second repository, using command create new serverless application, add two endpoint with two lambdas in new repo, and after that integrate their both. 
![image](https://user-images.githubusercontent.com/73526360/221539642-9fc233b8-b111-47c7-824b-ad229c29ad64.png)


Also could you provide with me link to your cloudfront distribution where I can see this integration.
![image](https://user-images.githubusercontent.com/73526360/221540064-e402a71f-7bfb-46dc-88a7-f8369a6a172a.png)
I tried to take a look to your first PR and try open link from their, but it doesn't load your mock data.

Reply:
after this task we will have two repositories, one for FE- this is fork from the first task. The second one is the backend, which has been created from this task.

Reply:
Regarding two repo. I decided to do it in one but split it into two different folders, a mono repo kind of. I do not see any problem doing it in one repo instead of two different ones. It is just more convenient for me and makes no difference. If you want, you can put fewer points in the final evaluation.

Reply:
No problem, as you wish, I think it's not a problem)